mlflow/server/js/src/components/tables/EditableFormTable.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { Table, Input, Form, Icon } from 'antd';
+import PropTypes from 'prop-types';
+
+import './EditableFormTable.css';
+
+const EditableContext = React.createContext();
+
+class EditableCell extends React.Component {
+  static propTypes = {
+    editing: PropTypes.bool,
+    dataIndex: PropTypes.string,
+    title: PropTypes.string,
+    record: PropTypes.object,
+    index: PropTypes.number,
+    children: PropTypes.oneOfType([
+      PropTypes.object,
+      PropTypes.array,
+    ]),
+  };
+
+  static defaultProps = {
+    style: {},
+  };
+
+  render() {
+    const { editing, dataIndex, title, record, children, ...restProps } = this.props;
+    return (
+      <EditableContext.Consumer>
+        {({ getFieldDecorator }) => (
+          <td {...restProps} className={editing ? 'editing-cell' : ''}>
+            {editing ? (
+              <Form.Item style={{ margin: 0 }}>
+                {getFieldDecorator(dataIndex, {
+                  rules: [
+                    {
+                      required: true,
+                      message: `Please Input ${title}!`,
```

Comment:
Maybe change this to `${title} is required` to match what we have for tag edit? Otherwise there's this discrepancy:
![image](https://user-images.githubusercontent.com/2358483/58988961-b5b72180-8797-11e9-9e7a-8838edef6966.png)

Reply:
Adjusted accordingly.